api/src/hooks/profile.hooks.test.ts

```diff
@@ -0,0 +1,266 @@
+import { describe, it, expect } from 'vitest'
+import {
+  checkUserHasProfile,
+  checkProviderInternal,
```

Comment:
```suggestion
  checkProviderIsInternal,
```
To keep the same naming logic between all this access rights related functions :slightly_smiling_face:

Reply:
done